lib/debug/model.dart

```diff
@@ -33,9 +34,11 @@ abstract class DebugConnection {
 // TODO: Add an IsolateState class.
 
 /// A representation of a VM Isolate.
-abstract class DebugIsolate {
+abstract class DebugIsolate extends MItem {
```

Comment:
I haven't looked closely into where you're using these now, but this file previously had a separation of concerns between non-ui (model) and UI code. I think it's still likely worth keeping this separation.